crates/router/src/core/payment_methods/transformers.rs

```diff
@@ -74,6 +74,58 @@ pub struct DeleteCardResp {
     pub error_code: Option<String>,
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AddCardRequest<'a> {
+    pub card_number: Secret<String, pii::CardNumber>,
+    pub customer_id: &'a str,
+    pub card_exp_month: Secret<String>,
+    pub card_exp_year: Secret<String>,
+    pub merchant_id: &'a str,
+    pub email_address: Option<Secret<String, pii::Email>>,
+    pub name_on_card: Option<Secret<String>>,
+    pub nickname: Option<String>,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AddCardResponse {
+    pub card_id: String,
+    pub external_id: String,
+    pub card_fingerprint: Secret<String>,
+    pub card_global_fingerprint: Secret<String>,
+    #[serde(rename = "merchant_id")]
```

Comment:
why `merchant_id` is renamed with `merchant_id`?

Reply:
I'll remove it, it was initially present, I didn't notice it while adding the code back